diff --git a/node_modules/react-native-paper/src/components/TextInput/TextInputFlat.tsx b/node_modules/react-native-paper/src/components/TextInput/TextInputFlat.tsx
index 2420dc4..8dd0e8c 100644
--- a/node_modules/react-native-paper/src/components/TextInput/TextInputFlat.tsx
+++ b/node_modules/react-native-paper/src/components/TextInput/TextInputFlat.tsx
@@ -424,7 +424,7 @@ const Underline = ({
 
   if (error) backgroundColor = colors?.error;
 
-  const activeScale = isV3 ? 2 : 1;
+  const activeScale = isV3 ? 0.5 : 1;
 
   return (
     <Animated.View
@@ -459,7 +459,7 @@ const styles = StyleSheet.create({
     left: 0,
     right: 0,
     bottom: 0,
-    height: 2,
+    height: 0.2,
     zIndex: 1,
   },
   md3Underline: {
